site/class-1-bar/maaikevc/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<!--Based on https://bl.ocks.org/mbostock/3885304 by Mike Bostock-->
+<meta charset="utf-8">
+<title>@maaikevc</title>
+
+<link rel="stylesheet" type="text/css" href="index.css">
+
+<body>
+<h1>De meest voorkomende uitroepen en kreten in het Nederlands volgens drs. M van Cruchten</h1>
+</body>
+
+<svg width="960" height="500"></svg>
+<script src="https://d3js.org/d3.v4.min.js"></script>
+
+
+<script src="index.js"></script>
+<script>
```

Comment:
This lone `<script>` tag is invalid, it’s missing a closing tag! Could you remove the `<script>` tag here (line 17)?